### ISSUE TITLE
French demo section for page #15 not working as expected

### ISSUE THREAD

**Issue by gonzalo-roberto-diaz** [angular-translate/angular-translate#1120]
https://angular-translate.github.io/docs/fr/#/guide/15_custom-interpolators

The "demo" section only shows this "--", compare with its English counterpart.

**Comment by tspaeth** [comment]
Yup. Due to missing messageformat update I guess.
Probably counts also for UK and CN?
```
ReferenceError: Locale fr could not be loaded: require is not defined
    at Function.MessageFormat.loadLocale (https://cdn.rawgit.com/SlexAxton/messageformat.js/0.2.2/messageformat.js:48:20)
```

**Comment by knalli** [comment]
Yeah, typo only in French version.

Fixed with 806b3b0, available with the next release.
